src/db.c

```diff
@@ -920,7 +920,8 @@ long long getObjectTypeByName(char *name) {
         }
     }
 
-    moduleType *mt = moduleTypeLookupModuleByName(name);
+    /* TODO: use moduleTypeLookupModuleByName instead in redis 8.0 */
```

Comment:
i don't think we'll want to remove it.
it breaks backwards compatibility and we don't gain anything.
let's drop the comment.

Reply:
and also the comments in the test.